packages/generator-react-server/generators/app/templates/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+services:
+  react:
```

Comment:
Is this `react` just a label for the service?

If so, could we make it `react-server`, or `react_server` if dashes aren't allowed?